onetl/connection/db_connection/kafka/connection.py

```diff
@@ -35,6 +35,56 @@
 log = logging.getLogger(__name__)
 
 
+PROHIBITED_OPTIONS = frozenset(
+    (
+        "assign",
+        "subscribe",
+        "subscribePattern",
+        "startingOffsets",
+        "startingOffsetsByTimestamp",
+        "startingTimestamp",
+        "endingOffsets",
+        "endingOffsetsByTimestamp",
+        "endingOffsets",
+        "startingOffsetsByTimestampStrategy",
+        "kafka.bootstrap.servers",
+        "kafka.group.id",
```

Comment:
```suggestion
        "kafka.*",
```
Любые опции для управления параметрами подключения к кафке должны задаваться на уровне самого коннектора, а не переопределяться при чтении из топика